src/components/MessageReviewPanel/MessagesList.tsx

```diff
@@ -1,24 +1,38 @@
 import { Layout } from "antd";
-import React from "react";
+import React, { useEffect, useRef } from "react";
 import { getMessageSenderType } from "src/utils/Message";
 import MessageCard, { TMessageItem } from "./MessageCard";
 
 interface Props {
   messages: TMessageItem[];
+  initialFocus?: TMessageItem;
 }
 
-const MessagesList = ({ messages }: Props) => {
+const MessagesList = ({ messages, initialFocus }: Props) => {
+  const initialFocusRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!initialFocusRef || !initialFocusRef.current) return;
```

Comment:
i don't think the ref container itself can be missing

```suggestion
    if (!initialFocusRef.current) return;
```